### ISSUE TITLE
Is digital ocean working?

### ISSUE THREAD

**Issue by blackhattristar** [AHAAAAAAA/PokemonGo-Map#3176]
I know Pokémon blocked access to the api via cloud servers. Is digital ocean working?
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by hedqvist** [comment]
I've got it running on DigitalOcean atm, 30+ workers in hive against a single mysql.

**Comment by blackhattristar** [comment]
@hedqvist So DigitalOcean wasn't affected by Pokémon denying access to their API via cloud servers?

**Comment by miaojs** [comment]
I'm blocked from ptc login servers on DO. Pgoapi was working fine tho.

**Comment by blackhattristar** [comment]
@miaojs Have you tried using DO with Google login?

**Comment by miaojs** [comment]
Nope. All my fake mapping accounts are ptc. Sorry. I guess I could create one to try if you really wanted to know and someone else doesn't chime in.

**Comment by blackhattristar** [comment]
You don't have to but if you do that would be useful information. I don't have a DO account or else I would try it. It's interesting though that DO still works. I wonder if Pokémon GO will catch on and block access coming from them as well.

**Comment by frazahod** [comment]
I am running just fine on DO with google account.

**Comment by blackhattristar** [comment]
@frazahod Great to hear!

**Comment by blackhattristar** [comment]
@hedqvist When you say "30+ workers in hive" what do you mean?

**Comment by hedqvist** [comment]
@blackhattristar  I mean that I got one session running the runserver.py with the -os option, just to show the map, then I have 30+ sessions of runserver.py with the -ns option to just added scanned data to MySQL.

So basiclly, 30+ scanners on selected areas (to map entire city), this gets added to MySQL and displayed via the runserver.py with the -os option.

**Comment by blackhattristar** [comment]
@hedqvist When I look at the Wiki for setting up DO, I see -H 0.0.0.0 and -P... I'm assuming I replace -H with my IP address and -P with 5000?

**Comment by miaojs** [comment]
You probably want 0.0.0.0 which will bind to all addresses. Port can be whatever you choose. 80 might be easiest so you don't have to specify a port number when accessing.

**Comment by blackhattristar** [comment]
Ok! @miaojs Thanks for the reply!

**Comment by blackhattristar** [comment]
If I add -a google in front of -u will that access the login via Google account?

**Comment by blackhattristar** [comment]
I've tried my IP with :80 and :5000 and I can't seem to get my IP to load in my browser after just setting up a droplet. Any help would be greatly appreciated. This is what I've set in my user data:

#!/bin/bash

apt-get -y update
apt-get -y install git nodejs npm python-pip nodejs-legacy python-dev
git clone -b develop https://github.com/AHAAAAAAA/PokemonGo-Map.git
cd PokemonGo-Map/
pip install -r requirements.txt
npm install
npm install -g grunt-cli
npm install node-sass
grunt build
python ./runserver.py -u myemail@gmail.com -p my password -st 10 -k mykey -l "lat, long” -H 0.0.0.0 -P 80 &

**Comment by miaojs** [comment]
Does the server run and search?

**Comment by blackhattristar** [comment]
@miaojs Do I tell that by looking at the Access Console? I type http://ipaddress into the address bar and it doesn't go anywhere or load anything.

**Comment by blackhattristar** [comment]
Theres no way it can be this difficult. I literally followed the Wiki instructions which are like two things to do just about and it doesn't work for me. I've got these droplets sitting there, active and I can't access the map. I don't know where it is.

**Comment by miaojs** [comment]
By "it doesn't go anywhere" do you mean it's a white screen or you get an error?

**Comment by blackhattristar** [comment]
@miaojs No white screen or error. It reroutes to like... dnsrsearch.com

**Comment by miaojs** [comment]
...you aren't literally typing in "iPad dress" right

**Comment by blackhattristar** [comment]
Right. I use the IP address that the droplet gives me.

**Comment by miaojs** [comment]
If it's redirecting you to something like that I'm not sure you've got the IP address typed right. Cam you double check?

**Comment by blackhattristar** [comment]
@miaojs Let's say the IP address is 137.68.16.24

I type http://137.68.16.24 into the browser url address bar.
